### PR TITLE
Fix README

### PR DESCRIPTION
**Pull request by gaokevin1**
## Description

Added table of contents to README.

## Must

- [ ] Tests
- [x] Documentation (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,13 @@ Install the package with:
 npm i --save @descope/node-sdk
 ```
 
-## Setup
+## Authentication Functions
 
-A Descope `Project ID` is required to initialize the SDK. Find it on the
+### Setup
+
+Before you can use authentication functions listed below, you must initialize a `DescopeClient`.
```
Comment (asafshen) [inline README.md:22]: I find this text a bit intimidating,  
I would just say something like
 - Make sure you have a Descope `Project ID`, you can find it on the ...
 - Create an sdk with the project ID
 
 ```typescript
const descopeClient = DescopeClient({ projectId: 'my-project-ID' });
```

**Review by asafshen — APPROVED** [review]
looks good! few minor comments, we can address later as well

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gaokevin1) [reply README.md]: I changed the name to descopeSdk, but I'm confused about what you mean by the createSdk methods. Isn't `descopeSdk` just an instance of the `DescopeClient` you import? Look at my changes and see if this is correct.

**Review by asafshen — DISMISSED** [review]

**Review by asafshen — DISMISSED** [review]

**Comment by dorsha** [comment]
@gaokevin1 need to make sure the semantics here aligned with all other sdks.

**Review by asafshen — APPROVED** [review]

**Pull request merged**